### PR TITLE
:book: Update consuming-existing-aws-infrastructure.md to provide context to where networkSpec belongs

### PR DESCRIPTION
**Pull request by cdenneen**
Provide context as to where the `networkSpec` is to be updated inside the `kind: AWSManagedControlPlane`

### CODE REVIEW

File: docs/book/src/topics/consuming-existing-aws-infrastructure.md [hunk 1]
```diff
@@ -34,6 +34,10 @@ Finally, if the controller manager isn't started with the `--configure-cloud-rou
 Specifying existing infrastructure for Cluster API to use takes place in the specification for the AWSCluster object. Specifically, you will need to add an entry with the VPC ID and the IDs of all applicable subnets into the `networkSpec` field. Here is an example:
 
 ```yaml
+apiVersion: controlplane.cluster.x-k8s.io/v1alpha3
```
Comment (richardcase) [inline docs/book/src/topics/consuming-existing-aws-infrastructure.md:37]: For EKS based clusters it lives on **AWSManagedControlPlane** but for EC2 based clusters it lives on **AWSCluster**. For each of these kinds it is in the spec.

So i think we should we should either:
1. Change this part `Specifying existing infrastructure for Cluster API to use takes place in the specification for the AWSCluster object`
2. Have a separate example or section for each cluster type

**Review by richardcase — CHANGES_REQUESTED** [review]

**Comment by cdenneen** [comment]
I signed it

**Comment by cdenneen** [comment]
I signed it

**Review by richardcase — APPROVED** [review]
/lgtm

**Pull request merged**